packages/localization/src/lib/resources.ts

```diff
@@ -292,9 +298,46 @@ export const resources = {
             Message: 'The problem will be solved soon',
           },
         },
-        Button: {
+        Buttons: {
           AddItem: 'Add item to library',
           AskManager: 'Ask a manager',
+          Claim: 'Claim a book',
+          Return: 'Return a book',
+          Extend: 'Extend claim period',
+        },
+        Messages: {
+          Errors: {
+            Claim: 'Something goes wrong with your claiming',
+            Extend: 'Something goes wrong with your extending',
+            Return: 'Something goes wrong with your returning',
+            ReportToManager: {
+              Title: 'We reported the problem to the manager',
+              Description: 'The problem will be solved soon',
+            },
+            Delete: {
+              Title: 'Warning',
+              Desctription: `The book {{title}} is now in the possession of a person {{userName}}. Are you sure you want to delete the book {{title}} from the library permanently?`,
+            },
+          },
+          Claim: {
+            Title: 'You have successfully claim the book',
```

Comment:
should be:
You have successfully claim**ed** the book

Reply:
fixed